model/opensearch.smithy

```diff
@@ -90,6 +90,7 @@ service OpenSearch {
         Count_Get_WithIndex,
         Count_Post,
         Count_Post_WithIndex,
+        CreateDistinguishedName,
```

Comment:
This should be the `UpdateDistinguishedNames` operation and in the correct spot on the list.